homeassistant/components/ezviz/config_flow.py

```diff
@@ -41,70 +37,98 @@ class EzvizConfigFlow(ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = CONN_CLASS_CLOUD_POLL
 
+    async def _validate_and_create(self, data):
+        """Validate the user input allows us to connect.
+
+        Data has the keys from DATA_SCHEMA with values provided by the user.
+        """
+
+        for entry in self.hass.config_entries.async_entries(DOMAIN):
+            if (
+                entry.data[CONF_USERNAME] == data[CONF_USERNAME]
+                and entry.data[CONF_PASSWORD] == data[CONF_PASSWORD]
```

Comment:
Should we really allow the same user to have different config entries with different passwords? Normally we just id on username. Then we can use our unique id config entries api:
https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-ids

Reply:
Updated config_flow with unique id based on username. Thank you for the info!